wakuv2/filter_manager.go

```diff
@@ -164,10 +164,16 @@ func (mgr *FilterManager) subscribeAndRunLoop(f filterConfig) {
 	}
 }
 
+func (mgr *FilterManager) networkChange() {
+	mgr.node.PingPeers() //Ping all peers to check if subscriptions are alive
```

Comment:
It would be great if we follow the convention of comments in status-go with `// ping...`.
There are other comments also not using this convention.